env-test.json

```diff
@@ -6,10 +6,13 @@
   "CONTENT_ADDRESS": "0xfC65F146B6a7ED57E802E5E1F9aD329F40D26d3B",
   "TOKEN_ADDRESS": "0xbCf7A789abC7f6D8aAEf378586326668757b7216",
   "NFT_ADDRESS": "0x06B65F0583F4D9E534fc96EA0D777fC7267a0bF3",
+  "WALLET_API_ENDPOINT": "https://test-api.testpeeranha.io/",
   "IPFS_API_URL_THE_GRAPH": "https://api.thegraph.com/ipfs/api/v0",
   "IPFS_API_URL": "https://ipfs.testpeeranha.io:5001",
+  "IPFS_API_URL_WEB3": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweGY0QzM1MDVGZjJmNDg0ZTAwZDM3Qzk0OEFkZmEzRWU5YTk5OTMwM2QiLCJpc3MiOiJ3ZWIzLXN0b3JhZ2UiLCJpYXQiOjE2NTU0ODk0OTI2NzgsIm5hbWUiOiJQZWVyYW5oYSBVSSJ9.lRqDmHTXLjSmWwWJWqjvaMhJPQP-fma-44EGRiClmTs",
   "POLYGON_CHILD_MANAGER_ADDRESS": "0xb5505a6d998549090530911180f38aC5130101c6",
   "MINTABLE_ERC20_PREDICATE_PROXY": "0x37c3bfC05d5ebF9EBb3FF80ce0bd0133Bf221BC8",
   "MINTABLE_ERC712_PREDICATE_PROXY": "0x56E14C4C1748a818a5564D33cF774c59EB3eDF59",
+  "SAVE_FILE_SERVICE": "save-file",
```

Comment:
No need to have an environment variable for that because it has the same value for all environments. Just hardcode this in the script.